docs/mac-broker-integration.md

```diff
@@ -0,0 +1,20 @@
+# macOS broker integration
```

Comment:
I would suggest we add code or docs in more universal way. This file's name and title could be "broker integration" because most of its content fit both Windows and Mac. We want the reader of this document to test both Mac and Windows, anyway.